src/client.rs

```diff
@@ -0,0 +1,38 @@
+use async_trait::async_trait;
+
+// This trait is defined because of mocking tests.
+#[async_trait]
+pub trait Clientable {
+    // see: https://github.com/Krout0n/mackerel-client-rs/blob/e49fc4fb535363db30aa876ccbe030ad7b59a8b0/src/host.rs#L129-L138
+    async fn post_metrics(
```

Comment:
See: https://crates.io/crates/async-trait
Trait-fns can't contain asyncness. So, async-trait is required. (rustc suggests that.)